docs/postgresql_setup.md

```diff
@@ -55,14 +55,14 @@ This step can be skipped if you always give the path of CodeChecker command.
 export PATH=~/codechecker_package/CodeChecker/bin:$PATH
 ~~~~~~
 
-Check a test project.
+Start web server to view the results.
 ~~~~~~{.sh}
-CodeChecker check --dbusername test_user --postgresql -n test_project_check -b "cd my_test_project && make clean && make"
+CodeChecker server --dbusername test_user --postgresql
 ~~~~~~
 
-Start web server to view the results.
+Check a test project.
```

Comment:
Nope. Since the "one storage and viewer server" update in 5.9, server needed to be started before a check. This update only changes so that check takes URL instead of host-port.